src/shared_modules/rsync/src/rsyncImplementation.cpp

```diff
@@ -399,7 +407,16 @@ void RSyncImplementation::sendAllData(const std::shared_ptr<DBSyncWrapper>& spDB
     {
         [&callbackWrapper, &messageCreator, &jsonSyncConfiguration] (const nlohmann::json & resultJSON)
         {
-            messageCreator->send(callbackWrapper, jsonSyncConfiguration, resultJSON);
+            const auto component { jsonSyncConfiguration.at("component").get_ref<const std::string&>() };
+
+            if (RSyncImplementation::instance().isComponentRegistered(component))
+            {
+                messageCreator->send(callbackWrapper, jsonSyncConfiguration, resultJSON);
+            }
+            else
+            {
+                throw std::runtime_error("Synchronization cancelled, component inactivated");
```

Comment:
This will create an ERROR message. From the user side, the concept of `component` is unrelated to inner implementation semantics.
I suggest to change this messages for a kindest one.

Reply:
This message is printed to stderr, via the component registration try-catch, it is not intended for users.